tf2_ros/include/tf2_ros/message_filter.h

```diff
@@ -686,7 +686,7 @@ class MessageFilter : public MessageFilterBase, public message_filters::SimpleFi
 
   static std::string stripSlash(const std::string & in)
   {
-    if (!in.empty() && (in [0] == '/')) {
+    if (!in.empty() && (in[0] == '/')) {
```

Comment:
@audrow So the issue seems to be that the linter in the PR job wants this to be the old code, that is `in [0] == '/'`.  Does the new linter want it to be without the space?